packages/core/src/components/Tabs/Tab.tsx

```diff
@@ -163,11 +187,40 @@ export default withStyles(({ color, font, pattern, unit, ui, transition }) => ({
     borderTopRightRadius: ui.borderRadius,
   },
 
-  tabButton_small: {
-    ...font.textSmall,
+  tabButton_secondary: {
+    ...pattern.regularButton,
+    paddingTop: unit / 2,
+    paddingBottom: (unit / 8) * 5,
+    fontWeight: 'normal',
+    justifyContent: 'center',
+    border: `${ui.borderWidth}px solid ${color.clear}`,
+    backgroundColor: color.clear,
+    borderRadius: ui.borderRadius,
+    ':hover': {
+      borderColor: color.accent.borderHover,
+      backgroundColor: color.accent.bgHover,
```

Comment:
the bg color on hover for this should be one darker, so `color.core.neutral[1]` (since bgHover is `neutral[0]` to be consistent with hover patterns on buttons.

Reply:
moving convo to single thread